components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -1472,6 +1477,64 @@ private void updateLocalAndOutboundAuthenticationConfiguration(int applicationId
         }
     }
 
+    /**
+     * Updates the application role mapping configuration.
+     *
+     * @param serviceProvider ServiceProvider to be updated.
+     * @param applicationRoleMappingConfigs Application role mapping configurations.
+     * @param connection Connection to the database.
+     * @throws SQLException SQLException when updating application role mapping configuration.
+     */
+    private void updateAppRoleMappingConfiguration(ServiceProvider serviceProvider,
+                                                   AppRoleMappingConfig[] applicationRoleMappingConfigs,
+                                                   Connection connection)
+            throws SQLException {
```

Comment:
```suggestion
    private void updateAppRoleMappingConfiguration(ServiceProvider serviceProvider,
                                                   AppRoleMappingConfig[] applicationRoleMappingConfigs,
                                                   Connection connection) throws SQLException {
```

Reply:
Addressed in 4cdf9672bdd6a4d6060995024d14bd9dc79a1168